linode/manager
# test: [M3-6168] - Added Service Transfer E2E test for Managed users

## Description 📝

Add a test case to confirm the user flow that occurs when a Managed user attempts to initiate a service transfer.

Managed users are not able to initiate service transfers, and this test should confirm that if a Managed user attempts to initiate a transfer, they are shown an error message and the flow fails gracefully.

## Preview 📷


![Screen Shot 2023-03-01 at 4 54 01 PM](https://user-images.githubusercontent.com/126020611/222273860-bdb9387f-8943-415d-9677-74bd1e913748.jpg)

## How to test 🧪

**What are the steps to reproduce the issue or verify the changes?**
1. Set up the Linode.
2. Navigate to Service Transfer landing page with managed enable.
3. Make a service transfer.
4. An error message "You cannot initiate transfers with Managed enabled." is shown.

**How do I run relevant unit or e2e tests?**
New test case is added to the existing test file.
